serialization-jackson/src/main/resources/META-INF/native-image/org.apache.pekko/pekko-serialization-jackson/reflect-config.json

```diff
@@ -0,0 +1,234 @@
+[
```

Comment:
pekko-serialization-jackson is configurable and users can choose to add extra Jackson modules - what happens in that case (in terms of the native image)? Will those users have to create their own reflect-config.json.